stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -216,11 +224,11 @@ type RequestConfig record {|
     http:CredentialBearer credentialBearer;
 |};
 
-# Process auth token for OAuth2 at token generation.
+# Generates the OAuth2 token.
 #
 # + authConfig - OAuth2 configurations
 # + oauth2CacheEntry - OAuth2 cache entry
-# + return - Auth token or `Error` if the validation fails
+# + return - OAuth2 token or `oauth2:Error` if the validation fails
```

Comment:
```suggestion
# + return - OAuth2 token or `oauth2:Error` if the validation fails
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - OAuth2 token, or else `oauth2:Error` if the validation fails
```